### PR TITLE
Garzon/group enhancement navigation

### LINKED ISSUE #29: Navigation component, Therapist Profile, and Unhealthy Thinking pages
Need to create these pages and component.

### PR DESCRIPTION
**Pull request by amgarz0n**
Closes #29

### CODE REVIEW

File: client/src/components/SideDrawer/SideDrawer.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    max-width: 400px;
+    /* The z-index property specifies the stack order of an element. 
+    An element with greater stack order is always in front of an element with a lower stack order. 
+    Note: z-index only works on positioned elements (position: absolute, position: relative, position: fixed, or position: sticky). */
+    z-index: 200;
+    color: #EDCAA7;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    /* border: 5px solid gold; */
```
Comment (ErnieAtLYD) [inline client/src/components/SideDrawer/SideDrawer.css:18]: I'm not crazy about leaving in commented out CSS but this isn't a deal-killer from accepting the PR.

File: client/src/components/SideDrawer/SideDrawer.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
+.side-drawer {
+    height: 100%;
+    background: white;
+    box-shadow: 1px 0px 10px rgba(0,0 , 0 , .3);
```
Comment (ErnieAtLYD) [inline client/src/components/SideDrawer/SideDrawer.css:4]: I would do `shadow: 1px 0 10px rgba(0, 0, 0, 0.3);` instead. Better safe than sorry. By the way, `0` is fine, no need for `0px`.

File: client/src/components/SideDrawer/SideDrawer.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            <Nav className="bottom-nav-items" defaultActiveKey="/" as="ul">
+                <Nav.Item as="li">
+                    <Link to="/about"><img className="nav-logo-icon" src={LogoIcon} alt="Mind Passage About Us" /></Link>
+                </Nav.Item>
+
+                <Nav.Item as="li">
+                    <Link to="/about">About Us</Link>
+                </Nav.Item>
+            </Nav>
+        </div >
```
Comment (ErnieAtLYD) [inline client/src/components/SideDrawer/SideDrawer.jsx:41]: `</div>`, no space at the end. #BecauseIAmOCD

File: client/src/components/Toolbar/Toolbar.css [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+
+body{ 
+    font-family: 'ITC Avant Garde Gothic Std Extra Light', sans-serif;
+}
+
+.nav-logo-icon img {
+    width: 10rem;
+    height: auto;
+    margin-left: .5rem;
```
Comment (ErnieAtLYD) [inline client/src/components/Toolbar/Toolbar.css:14]: `0.5rem`. I'm not 100% sure about this but Heroku may be a jerk if it doesn't have a 0 beforehand.

**Comment by ErnieAtLYD** [comment]
@amgarz0n also - smaller PRs more often > one large PR.

File: client/src/components/SideDrawer/SideDrawer.css (hunk 1 above)
Comment (amgarz0n) [reply client/src/components/SideDrawer/SideDrawer.css:18]: Thank you Ernie. I used those for guidelines, but I will delete the commented out CSS and push to branch again.

**Pull request merged**